java/xgboost4j/src/main/java/org/dmlc/xgboost4j/util/TransferUtil.java

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright (c) 2014 by Contributors 
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+    
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+package org.dmlc.xgboost4j.util;
+
+import org.dmlc.xgboost4j.DMatrix;
+
+/**
+ *
```

Comment:
this util seems to be private and should be invisible to the user. Consider put the two functions in private DMatrix or Trainer?